.storybook/preview-head.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Second time I've seen this sneaky new line <img src="https://media3.giphy.com/media/3oEjI789af0AVurF60/giphy.gif"/>

Reply:
Yea seems to get generated from running `yarn storybook` 👽